AnagramValidityChecker.java

```diff
@@ -0,0 +1,83 @@
+package CodeUAss1Package;
+
+import java.util.HashMap;
+
+public class AnagramValidityChecker
+{
+    /* Given two strings, determine if one is an anagram of the other.
+   Two sentences are anagrams if each word in the resulting
+    sentence is an anagram of one of the words in the original sentence.*/
+    public static boolean isSentenceAnagram(String sentence1, String sentence2, boolean isCaseSensitive)
+    {
+        // split the string to the words they are containing
+        String[] wordsOfSentence1 = sentence1.split(" ");
+        String[] wordsOfSentence2 = sentence2.split(" ");
+
+        if(wordsOfSentence1.length != wordsOfSentence2.length) return false;

```

Comment:
It is much better to always stick to the same style of formatting, no matter if the block following `if` (or `for` etc) is one or more than one line.

With formatting like:
```
if (condition) {
  action();
}
```
it is immediately clear what code will be executed if condition is true.

Without braces and with code in a single line the pattern is broken and suddenly person who reads code needs to spend brain cycles on actively parsing code, for example to detect possible bugs like:
```
if (condition) action1(); action2();
```

[Serious bugs](https://www.imperialviolet.org/2014/02/22/applebug.html) were introduced in production code and lack of braces contributed to missing them in review.